app/models/merchant.rb

```diff
@@ -0,0 +1,5 @@
+class Merchant < ApplicationRecord
+  has_many :items, dependent: :destroy
```

Comment:
This is definitely a good option for the relationship our tables currently have. Since Merchants have many items, when we delete a merchant, all of their items will also get destroyed. This gets a bit trickier when we have a many to many relationship. For example, what if items could belong to many merchants? Would you want the item to go away when just want of the merchants was destroyed? ... There's ways around this, but it's just something to think about when using this.